client/src/components/ImageList.jsx

```diff
@@ -1,78 +1,33 @@
-import React, {useState, useEffect, useRef} from 'react'
-import Image from './Image.jsx'
+import React from 'react'
 import styled from 'styled-components'
-import {LeftArrow, RightArrow, Pictures, Slider} from './styled.js'
-
-
-const ImageList = (props) => {
-const [pic, setPic] = useState(-50)
-const length = props.data.length;
-const [slide, setSlide] = useState(true)
-const [prev, setPrev] = useState(0);
-
-// const usePrevious = (value) => {
-//   const ref = useRef();
-
-//   useEffect(() => {
-//     ref.current = value;
-//   });
-
-//   return ref.current;
-// }
-
-// const previous = usePrevious(pic);
-
-// useEffect(() => {
-//   console.log('useEFFECT', pic)
-
-//   if(pic === 0) {
-//     setSlide(false)
-//   } else {
-//     setSlide(true)
-//   }
-// });
-
-const nextSlide = () => {
-  pic === ((length * -100) -50)? setPic(-50) : setPic(pic => pic-100);
-
-}
-
-const prevSlide = () => {
-  pic === 50 ? setPic((-100 * (length)) -50) : setPic(pic => pic + 100)
-
 
+const ImageWrapper = styled.div`
+
+`;
+const ImageSlider = styled.div`
+display: flex;
+flex-wrap: none;
+`;
+const Image = styled.img`
+`;
+const Arrow = styled.div`
+border-radius: 50%;
+color: black;
+background-color: white;
+`;
+
+const ImageList = ({data, active}) => {
+  return (
+    <ImageWrapper>
+      <Arrow>&#60</Arrow>
+      <ImageSlider>
+        {data.map((url, i) => {
+          return (<Image src={url} alt="Image" />)
```

Comment:
Do you have to add a 'key'? So you don't get the annoying warning of unique key?

Reply:
i very much do, thank you